Duplicati/UnitTest/RandomErrorBackend.cs

```diff
@@ -46,12 +48,12 @@ private void ThrowErrorRandom()
                 throw new Exception("Random upload failure");
         }
         #region IStreamingBackend implementation
-        public void Put(string remotename, Stream stream)
+        public async Task Put(string remotename, Stream stream, CancellationToken cancelToken)

```

Comment:
Is there a reason why some implementations of `Put` use `async/await`, while others do not?